R/exclude.R

```diff
@@ -237,9 +236,10 @@ normalize_exclusions <- function(x, normalize_path = TRUE,
 
     if (any(bad)) {
       stop("Full file exclusions must be character vectors of length 1. items: ",
-           toString(which(bad)),
-           " are not!",
-           call. = FALSE)
+        toString(which(bad)),
```

Comment:
I prefer the first argument bumped to a new kind as well

Reply:
Sorry, not sure what you mean by "bump" here.

Currently, it is:

```r
    if (any(bad)) {
      stop("Full file exclusions must be character vectors of length 1. items: ",
        toString(which(bad)),
        " are not!",
        call. = FALSE
      )
    }
```

And you'd like it to be formatted as the following?

```r
    if (any(bad)) {
      stop(
        "Full file exclusions must be character vectors of length 1. items: ",
        toString(which(bad)),
        " are not!",
        call. = FALSE
      )
    }
```

Reply:
yep that's what I mean